.github/workflows/fuzzydata-test.yml

```diff
@@ -0,0 +1,33 @@
+name: fuzzy
+on:
+  pull_request:
+jobs:
+  test-fuzzydata:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: modin
+          environment-file: environment-dev.yml
+          python-version: 3.8
+          channel-priority: strict
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+      - name: Conda environment
+        run: |
+          conda info
+          conda list
+      - uses: BSFishy/pip-action@v1
+        with:
+          packages: fuzzydata>=0.0.4
+      - run: python -m pytest modin/experimental/fuzzydata/test/test_fuzzydata.py -Wignore::UserWarning
+      - uses: actions/upload-artifact@v3
```

Comment:
I wonder if it is possible to see an uploaded artifact? Should we add `actions/download-artifact@v3` here too?

Reply:
Not sure i understand, the `action/upload-artifact@v3` allows the artifacts to stored and accesible within the Github actions tab. 

<img width="624" alt="image" src="https://user-images.githubusercontent.com/7931270/173253632-c917b791-c646-49f4-afc9-b4dd22934ee2.png">

You can visit my branch to see a sample artifact ZIP file here
https://github.com/suhailrehman/modin/actions/runs/2464078675

Reply:
I see, thanks! No need for actions/download-artifact@v3. I am also wondering how the performance is measured? Is there any documentation on this?

Reply:
Got it, thanks!

However, I think a one-shot time measurement is not the best way to check the performance. This should probably be considered in future fuzzydata releases if that makes sense to you.

Reply:
Yes! Should be an easy addition, can run the same op a couple of times and average to get a performance measurement. I'll follow the general guidelines that modin currently uses for its performance benchmarking

I don't expect this current integration to be perfect by any means, but hopefully, this conversation and your future thoughts on fuzzydata as you inspect its outputs can steer it towards being a useful system for your testing.